tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/flat-config/test-suite-a.html

```diff
@@ -241,11 +241,11 @@ <h3 id="testinterface-remarks">Remarks</h3>
           <p>Here are some remarks about the interface</p>
         </section>
         <section>
-          <h3>Construct Signatures</h3>
+          <h3>Constructors</h3>
```

Comment:
"Construct Signature" is not particularly useful to the end-user. It's a signature for a constructors. We already label interface methods (which are "method signatures" at the AST level) as "methods". This seems like a semantic improvement.